crates/ruma-common/src/events/room/encrypted/relation_serde.rs

```diff
@@ -92,12 +90,6 @@ struct RelatesToJsonRepr {
     relation: Option<RelationJsonRepr>,
 }
 
-impl RelatesToJsonRepr {
-    fn is_empty(&self) -> bool {
-        self.in_reply_to.is_none() && self.relation.is_none()
-    }
-}
```

Comment:
So I guess it's now possible that we serialize `"m.relates_to": {}`? Maybe not a big issue.

Reply:
The only case that would happen is with a custom relation, because we don't have any data. Which is what #1581 tries to solve :slightly_smiling_face:.